azure-pipelines/download_install_qt.ps1

```diff
@@ -21,5 +21,5 @@ Write-Output 'Installed QLI Installer Dependencies'
 Write-Output 'Starting QT Installer'
 $Env:QLI_OUT_DIR = ".\deps\Qt\Qt$qt_version"
 $Env:QLI_BASE_URL = "https://download.qt.io/online/qtsdkrepository/"
-python .\deps\qli-installer\qli-installer.py $qt_version windows desktop win64_msvc2017_64
+python .\deps\qli-installer\qli-installer.py $qt_version windows desktop win64_msvc2019_64
```

Comment:
Is this needed for Qt version change?

Reply:
This comes from upstream. For whatever reason 5.15.0 has the msvc_2017 version but 5.15.1+ do not. 

The msvc2019 change passed build on the Azure VS2017 builder though.

Reply:
The only thing I see with this is that we drop support for older platforms. I'd like to see conditionals.

Reply:
Conditionals are tough since your release builds only come from one place. I can roll it back to 5.15.0 if you're nervous about the msvc2019 part.